core/services/job/models.go

```diff
@@ -311,6 +311,7 @@ type OCR2OracleSpec struct {
 	ContractID                        string          `toml:"contractID"`
 	Relay                             relay.Network   `toml:"relay"`
 	RelayConfig                       JSONConfig      `toml:"relayConfig"`
+	RelayConfigMercuryConfig          JSONConfig      `toml:"relayConfigMercuryConfig"`
```

Comment:
do we need a separate element for this? can it be merged into RelayConfig?

Reply:
I think we can keep it separate as it is another toml configuration `[relayConfig.MercuryConfig]`